### PR TITLE
[plugin-web-app] Unify syntax of step saving number of opened tabs

### PR DESCRIPTION
**Pull request by vkepin**

### CODE REVIEW

File: vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/SetVariableSteps.java [hunk 1]
```diff
@@ -135,13 +135,13 @@ public void savePathFromUrl(Set<VariableScope> scopes, String variable)
      * <li><b>STORY</b> - the variable will be available within the whole story,
      * <li><b>NEXT_BATCHES</b> - the variable will be available starting from next batch
      * </ul>
-     * @param variable A name under which the value should be saved
+     * @param variableName A name under which the value should be saved
      */
-    @When("I get the number of open windows and set it to the $scopes variable '$variable'")
-    public void saveNumberOfOpenWindow(Set<VariableScope> scopes, String variable)
+    @When("I save number of open tabs to $scopes variable `$variableName`")
```
[the remark this answers is not in the document]
Comment (vkepin) [reply vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/SetVariableSteps.java:140]: maybe remove 'open' at all? -> I save number of tabs to $scopes variable `$variableName` ?

Comment (valfirst) [inline vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/SetVariableSteps.java:140]: I propose to do renaming (`tab` -> `window`) for all steps and methods at once

[the remark this answers is not in the document]
Comment (valfirst) [reply vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/SetVariableSteps.java:140]: let's keep `open tabs`

**Review by valfirst — CHANGES_REQUESTED** [review]

File: docs/modules/plugins/pages/plugin-web-app.adoc [hunk 2]
```diff
@@ -999,6 +999,24 @@ When I change context to element located `xpath(//table[1])`
 When I save table to SCENARIO variable `table`
 ----
 
+==== Save a number of opened tabs to the variable
```
Comment (valfirst) [inline docs/modules/plugins/pages/plugin-web-app.adoc:1002]:
```suggestion
==== Save the number of opened tabs
```

File: docs/modules/plugins/pages/plugin-web-app.adoc [hunk 3]
```diff
@@ -999,6 +999,24 @@ When I change context to element located `xpath(//table[1])`
...
+
+This step allows you to save a number of currently opened tabs in the browser.
```
Comment (valfirst) [inline docs/modules/plugins/pages/plugin-web-app.adoc:1004]:
```suggestion
Saves the number of currently opened tabs in the browser to the variable.
```

File: docs/modules/plugins/pages/plugin-web-app.adoc [hunk 4]
```diff
@@ -999,6 +999,24 @@ When I change context to element located `xpath(//table[1])`
...
+
+[source,gherkin]
+----
+When I save number of open tabs to $scopes variable `$variableName`
+----
+
+* `$scopes` - xref:commons:variables.adoc#_scopes[The comma-separated set of the variables scopes].
+* `$variableName` - The variable name to refer to the variable value.
+
+.Save the first table on the page into a variable named 'table'
```
Comment (valfirst) [inline docs/modules/plugins/pages/plugin-web-app.adoc:1014]: need to be updated

File: vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/SetVariableSteps.java [hunk 5]
```diff
@@ -146,7 +146,7 @@ public void savePathFromUrl(Set<VariableScope> scopes, String variable)
     }
 
     /**
-     * Gets an expected <b>value</b> from the <b>number of open windows</b>
+     * Gets an expected <b>value</b> from the <b>number of open tabs</b>
      * and saves it into the <b>variable</b>
      * <p>
      * A <b>value</b> is everything after the last <b>/</b>. <br>
```
Comment (valfirst) [inline vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/SetVariableSteps.java:152]:
```suggestion

```

File: vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/SetVariableSteps.java [hunk 6]
```diff
@@ -146,7 +146,7 @@ public void savePathFromUrl(Set<VariableScope> scopes, String variable)
     }
 
     /**
-     * Gets an expected <b>value</b> from the <b>number of open windows</b>
+     * Gets an expected <b>value</b> from the <b>number of open tabs</b>
```
Comment (valfirst) [inline vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/SetVariableSteps.java:149]: need to be updated (just copy from the docs)

File: vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/SetVariableSteps.java [hunk 7]
```diff
@@ -158,13 +158,13 @@ public void savePathFromUrl(Set<VariableScope> scopes, String variable)
      * <li><b>STORY</b> - the variable will be available within the whole story,
      * <li><b>NEXT_BATCHES</b> - the variable will be available starting from next batch
      * </ul>
-     * @param variable A name under which the value should be saved
+     * @param variableName A name under which the value should be saved
```
Comment (valfirst) [inline vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/SetVariableSteps.java:161]: align with docs

**Review by valfirst — CHANGES_REQUESTED** [review]

**Pull request merged**